collector/netdev_linux.go

```diff
@@ -65,10 +67,13 @@ func parseNetDevStats(r io.Reader, ignore *regexp.Regexp) (map[string]map[string
 			continue
 		}
 		netDev[dev] = map[string]string{}
-		for i, v := range header {
-			netDev[dev]["receive_"+v] = parts[i+1]
-			netDev[dev]["transmit_"+v] = parts[i+1+len(header)]
-		}
+                for i := 0; i < len(header_recieve); i++ {
+                        netDev[dev]["receive_"+header_recieve[i]] = parts[i+1]
+                }
+
+                for i := 0; i < len(header_transmit); i++ {
+                        netDev[dev]["transmit_"+header_transmit[i]] = parts[i+1]
```

Comment:
The offset on `parts` here is incorrect.  It should be `parts[i+1+len(recieveHeader)]`.